cmd/lifecycle-oracle/README.md

```diff
@@ -0,0 +1,25 @@
+# `lifecycle-oracle`
+
+The release lifecycle oracle serves data about phases for versions of products. Currently one API is supported: `/api/phases/`. This will return a document of all phases for all versions for all products, e.g.:
+
+```yaml
+ocp: # a product
+  "4.7": # a version
+    previous: # all previous phases
+      - event: "open" # the event
+        when: "2020-05-17T14:47:58Z" # when it occurred, optional
+    next: # the next event
+      event: "feature-freeze"
+  "4.6":
+    previous:
+    - event: "generally-available"
+      when: "2020-04-17T14:47:58Z"
+    - event: "code-freeze"
+      when: "2020-03-17T14:47:58Z"
+    - event: "feature-freeze"
+      when: "2020-02-17T14:47:58Z"
+    - event: "open"
+      when: "2020-01-17T14:47:58Z"
```

Comment:
In the related PR for a config manager, Alvaro asked if we can assume this array to be sorted (and hence the first one  in the list is always the last event). We may want to document the assumtion about ordering (or the lack of it, if needed) here.